example/__init__.py

```diff
@@ -0,0 +1,58 @@
+import datetime
```

Comment:
Unfortunately public examples need some docstrings, at least a quick summary what is going on in here.